tests/test_batch_run.py

```diff
@@ -158,9 +196,81 @@ def test_batch_run():  # noqa: D103
             "AgentID": 3,
             "agent_id": 3,
             "agent_local": 250.0,
+            "seed": None,
         },
     ]
 
+    result = mesa.batch_run(MockModel, {}, number_processes=2, rng=[42, 31415])
+
+    # we use 2 processes, so we are not guaranteed the order of the return
+    result = sorted(result, key=lambda x: (x["RunId"], x["AgentID"]))
+
+    assert result == [
+        {
+            "RunId": 0,
+            "iteration": 0,
+            "Step": 1000,
+            "reported_model_param": 42,
+            "AgentID": 1,
+            "agent_id": 1,
+            "agent_local": 250.0,
+            "seed": 42,
```

Comment:
There seems to be a lot of duplicate information in this output format. Maybe we should clean this up (at some point)